pkg/utils/pods.go

```diff
@@ -24,6 +24,7 @@ func GetFenceAgentsRemediationPod(r client.Reader) (*corev1.Pod, error) {
 	podNamespace, err := GetDeploymentNamespace()
 	if err != nil {
 		logger.Error(err, "failed fetching FAR namespace")
+		return nil, err
```

Comment:
This does not seem just related to the test.
Maybe a specific commit (if not a PR) to highlight this change might be beneficial